src/main/java/Duke.java

```diff
@@ -6,5 +8,64 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        String dashes = "____________________________________________________________";
```

Comment:
Maybe this could be declared as constant or maybe could be refactored with some other print statements to a method.